content/en/platform/corda/1.6/cenm/release-notes.md

```diff
@@ -26,9 +26,37 @@ title: Release notes
 ### Upgraded dependencies
 
 This release includes the following dependency version changes between CENM 1.5.9 and 1.6:
-* H2 has been upgraded from 1.4.197 to 2.1.214.
-* Hibernate has been upgraded from 5.4.32.Final to 5.6.14.Final.
-* Hibernate Validator has been upgraded to 6.2.5.Final.
-* Liquibase has been upgraded from 3.6.3 to 4.20.0.
-* Apache Tomcat has been upgraded from 9.0.80 to 9.0.81.
-* Bouncy Castle has been upgraded from 1.68 to 1.75.
+* Hibernate has been upgraded from 5.4.32.Final to 5.6.14.Final
+* Hibernate Validator has been upgraded to 6.2.5.Final
+* Apache Tomcat has been upgraded from 9.0.80 to 9.0.81
+* Bouncy Castle has been upgraded from 1.68 to 1.75
+
+* H2 has been upgraded from 1.4.197 to 2.1.214
+
+  H2 database has been upgraded to version 2.2.224 primarily to address vulnerabilities reported in earlier versions of H2.
+  H2 is not a supported production database and should only be utilized for development and test purposes. For detailed information
+  regarding the differences between H2 version 1.4.197 used in 4.10 and below, and the new H2 version 2.2.224 implemented in 4.11,
```

Comment:
.. used in (CENM) 1.5.9 and below ...

*can omit '(CENM)' just for reference*

Reply:
Updated